test/it/parser/src/main/resources/case/ddl/alter-table.xml

```diff
@@ -1600,11 +1590,6 @@
     
     <alter-table sql-case-id="alter_table_modify_lob_nocache">
         <table name="employees" start-index="12" stop-index="20" />
-        <modify-column>
```

Comment:
Why remove `modify-column`?

Reply:
Define incorrect use cases, and these SQL statements should be parsed by this syntax rule
Ref: https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/ALTER-TABLE.html#GUID-552E7373-BF93-477D-9DA3-B2C9386F2877__I2104157

Reply:
> Define incorrect use cases, and these SQL statements should be parsed by this syntax rule Ref: https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/ALTER-TABLE.html#GUID-552E7373-BF93-477D-9DA3-B2C9386F2877__I2104157

Got it, thank you.